css/index.css

```diff
@@ -143,4 +143,207 @@ footer nav {
 footer nav a {
     color: white;
     text-decoration: none;
-}
\ No newline at end of file
+}
+
+nav .nav-top {
+    display: flex;
+    align-content: flex-end;
+    width: 800px;
+    margin: 0 0 20px 0;
+    justify-content: space-between;
+} 
+
+.nav-top a {
+    font-size: 15px;
+    color: grey;
+    text-decoration: none;    
+}
+
+.menu {
+    width: 75%;
```

Comment:
Nice job creating a menu class and flexing it. Good use of flexbox properties throughout.